index.html

```diff
@@ -14,6 +15,81 @@
 </head>
 
 <body>
+
+<header>
+  <nav>
+    <a id="services-a" href="services.html">Services</a>
+    <a href="#">Product</a>
+    <a href="#">Vision</a>
+    <a href="#">Features</a>
+    <a href="#">About</a>
+    <a href="#">Contact</a>
+  </nav>
+  <div class="logo">
+  <img  src="img/logo.png" alt="Great Idea! Company logo.">
+  </div>
+</header>
+
+<section class="top-section">
+  <div class="inner-top-section">
+    <h1>Innovation <br> On <br> Demand</h1>
+    <button class="get-started">Get Started</button>
+  </div>
+  <img class="first-img" src="img/header-img.png" alt="Image of a code snippet.">
+</section>
+
+<div class="container">
+  <section class="main">
+    <div class="top-half">
+      <div class="features">
+        <h2> Features </h2>
+        <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis. </p>
+      </div>
+
+      <div class="about">
+        <h2>About</h2>
+        <p> Aliquam elementum magna eros, ac posuere elvit tempus et. sSuspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </div>
+
+
+    <img class="second-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="bottom-half">
+      <div class="services">
+        <h2>Services</h2>
+        <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis. </p>
+      </div>
+
+      <div class="product">
+        <h2>Product</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div class="vision">
+        <h2>Vision</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </div>
+
+  </section>
+
+  <footer>
+    <h2>Contact</h2>
+    <address>
+      123 Way 456 Street <br>
+      Somewhere, USA <br> <br>
+      1 (888) 888-8888 <br> <br>
```

Comment:
```suggestion
      <a href="tel:18888888888">1 (888) 888-8888 </a><br> <br>
```
Wrapping this in an anchor tag will tell the browser that it is a phone number and make for a more semantic and better UI/UX. You can do something similar with the email address below this line